src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -159,63 +173,53 @@ public void FileConfig_Test2()
         public void GetEvalutedPropertyValue_NoException_ReturnsValue()
         {
             // Arrange
-            var dummyEngine = new DummyProjectEngine();
-
-            string suppliedPropertyName = null;
-            dummyEngine.TestOp = pn =>
-            {
-                suppliedPropertyName = pn;
-                return "propertyValue";
-            };
+            var settingsMock = new Mock<IVCRulePropertyStorage>();
+            settingsMock.Setup(x => x.GetEvaluatedPropertyValue("propertyName1"))
+                .Returns("propertyValue");
 
             // Act
             var result =
-                CFamilyHelper.FileConfig.GetPotentiallyUnsupportedPropertyValue(dummyEngine, "propertyName1",
+                CFamilyHelper.FileConfig.GetPotentiallyUnsupportedPropertyValue(settingsMock.Object, "propertyName1",
                     "default xxx");
 
             // Assert
             result.Should().Be("propertyValue");
-            suppliedPropertyName.Should().Be("propertyName1");
         }
 
         [TestMethod]
         public void GetEvalutedPropertyValue_Exception_ReturnsDefault()
```

Comment:
Test name is incorrect, should be something like 'GetPotentiallyUnsupportedPropertyValue_PropertyUnsupported_ReturnsDefaultValue'